src/ssiApi/apiAuth/apiAuth.ts

```diff
@@ -0,0 +1,29 @@
+import fetch from "node-fetch"
+import { IgenerateToken } from "./IAuth";
+export class ApiAuth {
+    private apiKey: string;
+    constructor(apiKey) {
+        if (!apiKey || apiKey.trim() === "") {
+            throw new Error("HID-SSI_SDK:: Error: Please Provide apiKey")
+        }
+        this.apiKey = apiKey
+    }
+    async generateAccessToken(): Promise<IgenerateToken> {
+        const studioApiUrl = "https://api.entity.hypersign.id/api/v1/app/oauth"
+        const headers = {
+            "X-Api-Secret-Key": this.apiKey,
+            "Origin": "https://entity.hypersign.id"
```

Comment:
Origin should not also be hardcoded, I guess you need to take this in constructor params

Reply:
No need of passing origin. Its working fine without passing origin also